internal/pubsub/drivertest/drivertest.go

```diff
@@ -53,12 +53,6 @@ func RunConformanceTests(t *testing.T, newHarness HarnessMaker) {
 	t.Run("TestSendReceive", func(t *testing.T) {
 		testSendReceive(t, newHarness)
 	})
-	t.Run("TestErrorOnSendToClosedTopic", func(t *testing.T) {
```

Comment:
Why are you deleting tests?

Reply:
I think jba is planning to getting rid of the `Close` methods, so I don't think it makes sense to keep tests that involve them. But maybe we want to wait until we actually remove those.

Reply:
I'll try bringing them back but I think there was a reason why I removed them.

Reply:
Done.